aws/aws-toolkit-vscode
# Fix case where Extension startup tries to load a profile and the config file does not exist

## Types of changes



- [ ] Infrastructure (change which improves the lifecycle management (CI/CD, build, package, deploy, lint, etc) of the application, but does not change functionality.)
- [ ] Technical debt (change which improves the maintainability of the codebase, but does not change functionality)
- [ ] Testing (change which modifies or adds test coverage, but does not change functionality)
- [ ] Documentation (change which modifies or adds documentation, but does not change functionality)

## Description

If you've previously connected to AWS with a credentials profile and closed VS Code, the next time you open VS Code it uses the same profile. On startup, if the user does not have a config file, a "file does not exist" error shows up. This change prevents the code from trying to load profiles from the (non-existent) config file.

## Motivation and Context



## Related Issue(s)



## Testing

* Connect to AWS with a profile, then close VS Code
* delete (or rename) .aws/config file
* Reopen VS Code, select AWS Explorer
* Extension will attempt to re-connect to AWS with the same profile
* expect no error message

## Screenshots (if appropriate)

## Checklist





## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
